api/src/dev/server/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine
+FROM alpine:3.7
```

Comment:
I especially appreciate pinning the version here! Thank you CH!

Reply:
+1, not doing this has bitten me before.